base/array.jl

```diff
@@ -1434,7 +1434,7 @@ end
 """
     sizehint!(s, n; first::Bool=false, shrink::Bool=true) -> s
 
-Suggest that collection `s` reserve capacity for at least `n` elements. That is, if
```

Comment:
I cannot agree with this change. Usually, it reserves space for more elements (even when shrinking), so the original texting is correct. For example for `Dict` it always allocates by the power of 2.